### PR TITLE
Update the migration docs

### PR DESCRIPTION
**Pull request by Bwko**
This obsoletes #42

### CODE REVIEW

File: content/doc/upgrade/from-gogs.en-us.md [hunk 1]
```diff
@@ -20,42 +20,57 @@ Gogs versions up to 0.9.97 can be smootlhy upgraded to Gitea.
...
-* cp -r gitea-v1.0.0/options gitea/options
-* cp -r gogs-repositories/ gitea-repositories
-* cp -r gogs-data/ gitea-data/
-* vim gitea/custom/conf/app.ini
-
-Change gogs specific information:
+* Download the file matching your platform from the [downloads page](https://dl.gitea.io/gitea)
+* Unzip at the desired install location
+* Copy Gogs `custom/conf` to Gitea
+* Rename `custom/conf` to `custom/options`
```
Comment (bkcsoft) [inline content/doc/upgrade/from-gogs.en-us.md]: This is wrong AFAIK

Comment (lunny) [reply content/doc/upgrade/from-gogs.en-us.md]: Yes. Remove this line.

**Comment by Bwko** [comment]
@lunny & @bkcsoft Fixed

**Comment by bkcsoft** [comment]
@Bwko you complete left this part out though...

> If you have custom content migrated from gogs/custom/ folder, like templates or localization files, you should manually merge these because these files changed on Gitea.

File: content/doc/upgrade/from-gogs.en-us.md [hunk 2]
```diff
@@ -20,42 +20,56 @@ Gogs versions up to 0.9.97 can be smootlhy upgraded to Gitea.
...
-* cp -r gogs gitea
-* If you have custom content migrated from gogs/custom/ folder, like templates or localization files, you should manually merge these because these files changed on Gitea.
-* cp -r gitea-v1.0.0/options gitea/options
-* cp -r gogs-repositories/ gitea-repositories
-* cp -r gogs-data/ gitea-data/
-* vim gitea/custom/conf/app.ini
-
-Change gogs specific information:
+* Download the file matching your platform from the [downloads page](https://dl.gitea.io/gitea)
+* Unzip at the desired install location
```
Comment (tboerger) [inline content/doc/upgrade/from-gogs.en-us.md]: It's not zipped

File: content/doc/upgrade/from-gogs.en-us.md [hunk 3]
```diff
@@ -20,42 +20,56 @@ Gogs versions up to 0.9.97 can be smootlhy upgraded to Gitea.
...
+* Copy Gogs `custom/conf` to Gitea
```
Comment (tboerger) [inline content/doc/upgrade/from-gogs.en-us.md]: And the stuff beside app.ini needs to be copied to options

**Comment by Bwko** [comment]
@tboerger & @bkcsoft Done (I hope :wink:)

File: content/doc/upgrade/from-gogs.en-us.md [hunk 4]
```diff
@@ -20,42 +20,58 @@ Gogs versions up to 0.9.97 can be smootlhy upgraded to Gitea.
...
-* cp -r gitea-v1.0.0/options gitea/options
-* cp -r gogs-repositories/ gitea-repositories
-* cp -r gogs-data/ gitea-data/
-* vim gitea/custom/conf/app.ini
-
-Change gogs specific information:
+* Download the file matching your platform from the [downloads page](https://dl.gitea.io/gitea)
+* Put the binary at the desired install location
+* Copy `gogs/custom/conf/app.ini` to `gitea/custom/conf/app.ini`
+* If you have custom content in `gogs/custom/` copy that to `gitea/custom/options`. This is used to override gitignores, labels, licenses, locales and readmes.
```
Comment (lunny) [inline content/doc/upgrade/from-gogs.en-us.md]: `gogs/custom/` -> `gitea/custom/`, no `options`?

Comment (Bwko) [reply content/doc/upgrade/from-gogs.en-us.md]: Gogs doesn't have a options folder

**Comment by lunny** [comment]
LGTM

File: content/doc/upgrade/from-gogs.en-us.md [hunk 5]
```diff
@@ -15,47 +15,63 @@ menu:
...
-* If you have custom content migrated from gogs/custom/ folder, like templates or localization files, you should manually merge these because these files changed on Gitea.
-* cp -r gitea-v1.0.0/options gitea/options
-* cp -r gogs-repositories/ gitea-repositories
-* cp -r gogs-data/ gitea-data/
-* vim gitea/custom/conf/app.ini
+* Create a Gogs backup with `gogs dump`. This creates `gogs-dump-[timestamp].zip` file containing all your Gogs data. 
+* Download the file matching your platform from the [downloads page](https://dl.gitea.io/gitea)
+* Put the binary at the desired install location
+* Copy `gogs/custom/conf/app.ini` to `gitea/custom/conf/app.ini`
+* If you have custom content in `gogs/custom/` copy that to `gitea/custom/options`. This is used to override gitignores, labels, licenses, locales and readmes.
```
Comment (tboerger) [inline content/doc/upgrade/from-gogs.en-us.md]: `gogs/custom/conf` got things like gitignores, licenses, labels and so on which needs to be copied to `gitea/custom/options`. Beside that `gogs/custom/` can also include `templates` and `public` which needs to be copied to `gitea/custom/`.

File: content/doc/upgrade/from-gogs.en-us.md [hunk 6]
```diff
@@ -15,47 +15,63 @@ menu:
...
-* [database]
-* PATH = /home/:USER/gogs/data/:DATABASE.db
-* [attachment]
-* PATH = /home/:USER/gogs-data/attachments
-* [picture]
-* AVATAR_UPLOAD_PATH = /home/:USER/gogs-data/avatars
-* [log]
-* ROOT_PATH = /home/:USER/gogs/log
+
+    [database]
```
Comment (tboerger) [inline content/doc/upgrade/from-gogs.en-us.md]: Just drop the whitepsace prefix and add ``` around it.

File: content/doc/upgrade/from-gogs.en-us.md [hunk 7]
```diff
@@ -15,47 +15,63 @@ menu:
...
-* [log]
-* ROOT_PATH = /home/:USER/gitea/log
-
-Verify working by running gitea
-
-* ~/gitea/gitea web
-* Enter Gitea admin panel on the UI, run `Rewrite '.ssh/authorized_keys' file` (caution: non-Gitea keys will be lost) and `Rewrite all update hook of repositories` (needed when custom config path is changed).
 
-Update the gitea-v1.0.0/scripts file for your specific distro and environment then copy.
+    [database]
```
Comment (tboerger) [inline content/doc/upgrade/from-gogs.en-us.md]: Same as above

File: content/doc/upgrade/from-gogs.en-us.md [hunk 8]
```diff
@@ -15,47 +15,63 @@ menu:
...
+### Add Gitea to startup on Unix
+Update the appropriate file from [gitea/scripts](https://github.com/go-gitea/gitea/tree/master/scripts) with the right environment variables.
+
+For distro's with systemd:
+
+* Copy the updated script to `/etc/systemd/system/gitea.service`
+* Add the service to the startup with: `sudo systemctl enable gitea`
+* Disable old gogs startup script: `sudo systemctl disable gogs`
+
+For distro's with open-rc:
```
Comment (tboerger) [inline content/doc/upgrade/from-gogs.en-us.md]: SysVinit instead of open-rc?

**Comment by tboerger** [comment]
LGTM

**Pull request merged**